### PR TITLE
Updates for Auth2 compatibility

### PR DESCRIPTION
**Pull request by msneddon**

### CODE REVIEW

File: lib/KBaseReport/KBaseReportImpl.pm.bak-2017-03-06-15-59-41 [hunk 1]
```diff
@@ -0,0 +1,1078 @@
+package KBaseReport::KBaseReportImpl;
```
Comment (rsutormin) [inline lib/KBaseReport/KBaseReportImpl.pm.bak-2017-03-06-15-59-41]: Is there any reason for committing this file.

File: test/KBaseReport_server_test.pl [hunk 2]
```diff
@@ -13,7 +13,7 @@
 my $config = new Config::Simple($config_file)->get_block('KBaseReport');
 my $ws_url = $config->{"workspace-url"};
 my $ws_name = undef;
-my $ws_client = new Bio::KBase::workspace::Client($ws_url,token => $token);
+my $ws_client = Workspace::WorkspaceClient->new($ws_url,token => $token);
 my $auth_token = Bio::KBase::AuthToken->new(token => $token, ignore_authrc => 1);
```
Comment (rsutormin) [inline test/KBaseReport_server_test.pl]: I think this line can not work properly. Why don't you refactor token validation similar to https://github.com/kbase/kb_sdk/blob/auth2/src/java/us/kbase/templates/module_test_perl_client.vm.properties#L22 ?

File: lib/DataFileUtil/DataFileUtilClient.pm [hunk 3]
```diff
@@ -0,0 +1,3352 @@
+package DataFileUtil::DataFileUtilClient;
```
Comment (rsutormin) [inline lib/DataFileUtil/DataFileUtilClient.pm:1]: I don't see you using DFU anywhere in this PR. Is that right?

**Review by rsutormin — CHANGES_REQUESTED** [review]

File: lib/KBaseReport/KBaseReportImpl.pm.bak-2017-03-06-15-59-41 (hunk 1 above)
Comment (msneddon) [reply lib/KBaseReport/KBaseReportImpl.pm.bak-2017-03-06-15-59-41]: yeah, let me clean up those files- there are several bak files in there from before too.

File: test/KBaseReport_server_test.pl (hunk 2 above)
Comment (msneddon) [reply test/KBaseReport_server_test.pl]: hmm- it worked enough to save to CI workspace, but I'll switch to do use the better pattern.

File: lib/DataFileUtil/DataFileUtilClient.pm (hunk 3 above)
Comment (msneddon) [reply lib/DataFileUtil/DataFileUtilClient.pm:1]: Right- I was going to switch, but it seemed to work without doing that.  That will be round 2.

**Comment by rsutormin** [comment]
@msneddon: PR looks good to me now. Let's merge it?

**Pull request merged**